cfme/tests/cloud/test_provisioning.py

```diff
@@ -95,9 +95,25 @@ def test_provision_from_template(request, setup_provider, provider, testing_inst
              num_sec=1000,
              delay=60,
              handle_exception=True)
-    if provider.type == 'gce' and current_version() >= "5.7":
-        soft_assert('Yes' in instance.get_detail(
-            properties=("Properties", "Preemptible")), "GCE Instance isn't Preemptible")
+    soft_assert(instance.does_vm_exist_on_provider(), "Instance wasn't provisioned")
+
+
+@pytest.mark.uncollectif(lambda provider: provider.type != 'gce')
+def test_gce_preemtible_provision(request, setup_provider, provider, testing_instance, soft_assert):
+    instance, inst_args = testing_instance
+    instance.create(**inst_args)
+    instance.wait_to_appear(timeout=800)
+    provider.refresh_provider_relationships()
```

Comment:
@lkhomenk This is not a blocker on this PR thugh, just flagging it for future work